test/lit/passes/instrument-memory.wast

```diff
@@ -406,6 +421,8 @@
   ;; CHECK-NEXT:  )
   ;; CHECK-NEXT: )
   (func $A (type $1)
+    (drop (memory.grow (i32.const 4)))
```

Comment:
Looks like this is still an issue - ping me for review when it is ready.

Reply:
Could you clarify what's the issue here? Is it about lack of comment? I don't mind adding it, but in that case should the comment be added to all the other instructions in this file, as we don't specify any filter so all the memory operations are instrumented.

Reply:
Sorry, I should have linked to my other comment that this was a followup to,

```


Maybe add this at the end, not the start? That would make the diff far shorter.

It might be even better in a new function, for easier reading.
```
([link](https://github.com/WebAssembly/binaryen/pull/7388/#discussion_r2027489976))

My concern is the location. By putting this new test at the top, look at how large the diff is. All the existing tests end up modified. I am suggesting you add the new stuff at the bottom, to minimize the diff.

Reply:
Oh you're right, updated that.